app/views/layouts/_header.html.erb

```diff
@@ -18,7 +18,7 @@
     <%= link_to t('menu.courses'), courses_path, class: 'btn btn-info' %>
     <%= link_to t('menu.hometasks'), '#', class: 'btn btn-info' %>
     <%= link_to t('menu.group'), groups_path, class: 'btn btn-info' %>
-    <%= link_to t('menu.subjects'), subjects_path, class: 'btn btn-info' %>
+    <%= link_to t('menu.timetable'), '#', class: 'btn btn-info' %>
```

Comment:
PR would be merged after Timetable implementation, when you can add timeslots_path here.

Reply:
@yasakov-e you already can add `timeslots_path`